src/common/paths/dbrps.yml

```diff
@@ -68,7 +72,7 @@ post:
     content:
       application/json:
         schema:
-          $ref: "../schemas/DBRP.yml"
+          $ref: "../schemas/DBRPCreate.yml"
```

Comment:
Trying to use the same schema for the create body and the response was problematic, so a separate schema was created.